### PR TITLE
Update committees emails and descriptions

### PR DESCRIPTION
**Pull request by pedrosino**
Changes the email for WDC and WRC to get consistent with the others.
Also basic information about WEC and WQAC.

### CODE REVIEW

File: WcaOnRails/db/migrate/20171219204656_update_committes_emails.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class UpdateCommittesEmails < ActiveRecord::Migration[5.1]
+  def change
+    execute <<-SQL
+      UPDATE teams set email = 'disciplinary@worldcubeassociation.org' where friendly_id = 'wdc';
+    SQL
+
+    execute <<-SQL
+      UPDATE teams set email = 'regulations@worldcubeassociation.org' where friendly_id = 'wrc';
```
Comment (jfly) [inline WcaOnRails/db/migrate/20171219204656_update_committes_emails.rb]: Is this also in teams.seeds.rb? If so, we should update that as well (or honestly, we could probably delete that file without it causing issue).

File: WcaOnRails/db/structure.sql [hunk 2]
```diff
@@ -1241,9 +1241,11 @@ INSERT INTO `schema_migrations` (version) VALUES
 ('20170823170616'),
 ('20170830140540'),
 ('20170831170616'),
+('20170930231828'),
```
Comment (jfly) [inline WcaOnRails/db/structure.sql]: Is this a phanton migration?

File: WcaOnRails/db/migrate/20171219204656_update_committes_emails.rb (hunk 1 above)
Comment (pedrosino) [reply WcaOnRails/db/migrate/20171219204656_update_committes_emails.rb]: LOL, two fails in 24h

File: WcaOnRails/db/structure.sql (hunk 2 above)
Comment (pedrosino) [reply WcaOnRails/db/structure.sql]: Yeah, it's from my delegates table work.

**Pull request merged**